nodedb.go

```diff
@@ -1007,6 +1020,7 @@ func (ndb *nodeDB) orphans() ([][]byte, error) {
 // Not efficient.
 // NOTE: DB cannot implement Size() because
 // mutations are not always synchronous.
+//
```

Comment:
Better to remove this line to avoid issues with merging/upstreaming.

Reply:
It's something that's added automatically when we have //nolint:unused